lib/views/physician/create_conversation.dart

```diff
@@ -0,0 +1,130 @@
+  String? desc = "";
+  ServiceType? serviceType;
+
+
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder<Map<String, dynamic>?>(
+      // future: retrieveAppointmentCreationInfo(),
+      future:
+          _prefs.getUid().then((uid) => retrieveChatCreationInfo(uid)),
+      builder: ((context, snapshot) {
+        // List of physician's patients
+        List<PatientID> list = snapshot.data == null
+            ? []
+            : snapshot.data?["patients"] as List<PatientID>;
+        return Stack(children: [
+          Padding(
+            padding: const EdgeInsets.all(16.0),
+            child: Container(
+              constraints: const BoxConstraints(maxWidth: 1000),
+              child: Form(
+                key: _formKey,
+                child: Column(
+                  children: [
+                    DropdownButtonFormField(
+                        validator: (value) => requiredValue(value),
+                        decoration: const InputDecoration(
+                          hintText: 'Patient',
+                          prefixIcon: Icon(Icons.person),
+                        ),
+                        items: list.map((e) {
+                          return DropdownMenuItem<PatientID>(
+                              value: e, child: Text(e.name));
+                        }).toList(),
+                        onChanged: (PatientID? value) {
+                          debugPrint("temmp");
+                        },
+                        onSaved: (value) => {patient = value},
+                        value: patient),
+                    gap(),
+                    TextFormField(
+                      keyboardType: TextInputType.multiline,
+                      validator: (value) => requiredValue(value),
+                      maxLines: null,
+                      minLines: 3,
+                      onSaved: (value) => {desc = value},
+                      decoration: const InputDecoration(
+                        hintText: 'initial message',
+                        prefixIcon: Icon(Icons.description),
+                      ),
+                    ),
+                    
+                    ElevatedButton(
+                        onPressed: () {
+                          _formKey.currentState?.save();
+                            Map<String, dynamic> payload = {
+                              "patient": patient!.name,
+                              "patient_id": patient!.id,
+                              "message": desc,
+                            };
+                            _prefs.getUid().then(
+                                (value) => createConversation(payload, value));
+                            // CreateConversation(payload);
+                            
+                            Navigator.pop(context);
+                          
+                        },
+                        child: const Text("Create Appointment"))
```

Comment:
Still says create appointment here, probably should be create chat or something.

Reply:
oops xd